scapy/contrib/gtp_v2.py

```diff
@@ -333,8 +355,8 @@ class ULI_SAI(ULI_Field):
     fields_desc = [
         gtp.TBCDByteField("MCC", "", 2),
         gtp.TBCDByteField("MNC", "", 1),
-        ShortField("LAC", 0),
-        ShortField("SAC", 0),
+        BitField("LAC", 0, 16),
```

Comment:
Why is this change needed?

Reply:
Because, I think this is wrongly encoded. As per 3gpp specs, the encoding is as below:
```
  |   | Bits |  
-- | -- | -- | --
  | Octets | 8 | 7 | 6 | 5 | 4 | 3 | 2 | 1 |  
  | b | MCC digit 2 | MCC digit 1 |  
  | b+1 | MNC digit 3 | MCC digit 3 |  
  | b+2 | MNC digit 2 | MNC digit 1 |  
  | b+3 to b+4 | Location Area Code (LAC) |  
  | b+5 to b+6 | Service Area Code (SAC) |  
  |   |   |   |   |   |   |   |   |   |  

The Location Area Code (LAC) consists of 2 octets. Bit 8 of Octet b+3 is the most significant bit and bit 1 of Octet b+4 the least significant bit. The coding of the location area code is the responsibility of each administration. Coding using full hexadecimal representation (binary, not ASCII encoding) shall be used.
The Service Area Code (SAC) consists of 2 octets. Bit 8 of Octet b+5 is the most significant bit and bit 1 of Octet b+6 the least significant bit. The SAC is defined by the operator. See 3GPP TS 23.003 [2] clause 12.5 for more information. 
```
So, LAC is 2 bytes and SAC is next 2 bytes.
Hence, I made a change to reflect its correct length to avoid wrong encoding & decoding.

Same comment applies for all these `ULI` fields.

Reply:
ShortField is 16 bits. Why is this change useful?
ShortField x 2 = bitField(16) x 2

Reply:
Oh, if ShortField is 2 bytes, then that works; I will back out my changes.
Sorry, I thought its 1 byte.